src/sparseml/pytorch/image_classification/utils/helpers.py

```diff
@@ -214,6 +217,53 @@ def _create_val_dataset_and_loader(
     return None, None  # val dataset not needed
 
 
+def _create_ffcv_train_and_val_loaders(args, image_size: int = 224):
+
+    warnings.warn(
+        "`ffcv` support is experimental and currently only supports "
+        "`ImageNet` based datasets."
+    )
+    train_dataset = DatasetRegistry.create(
+        args.dataset,
+        root=args.dataset_path,
+        train=True,
+        rand_trans=True,
+        image_size=image_size,
+        **args.dataset_kwargs,
+    )
+
+    val_dataset = DatasetRegistry.create(
+        args.dataset,
+        root=args.dataset_path,
+        train=False,
+        rand_trans=False,
+        image_size=image_size,
+        **args.dataset_kwargs,
+    )
+
+    if not (
+        isinstance(train_dataset, FFCVCompatibleDataset)
+        and isinstance(val_dataset, FFCVCompatibleDataset)
+    ):
+        raise ValueError("The Datasets have to be FFCV compatible for using ffcv")
+
+    train_loader = train_dataset.get_ffcv_loader(
+        write_path="data/ffcv-train.beton",
```

Comment:
hardcoded value here

Reply:
defaulted to using `default_dataset_path(f"{args.dataset}_ffcv_train.beton")`